app/migrations/20190604123759-create-file.js

```diff
@@ -0,0 +1,41 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('files', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      path: {
+        field: 'path',
+        type: Sequelize.STRING
+      },
+      entityType: {
+        field: 'entity_type',
+        type: Sequelize.STRING
+      },
+      entityId: {
+        field: 'entity_id',
+        type: Sequelize.INTEGER,
+        onDelete: 'SET NULL',
+        references: {
+          model: 'companies',
```

Comment:
This field doesn't references only the company. Together with entity type it can reference any entity.